moduleroot/.travis.yml.erb

```
@@ -26,6 +26,8 @@ before_install:
 script:
   - 'bundle exec rake $CHECK'
 matrix:
+  allow_failures:
+    - env: BEAKER_PUPPET_COLLECTION=puppet6-nightly
```

I'd prefer to start without this so we can actually see it fail. I know I wont open Travis to see if those failed and I think it'll still be silently ignored